test/fit_tests/deploy/rackhd_source_install.py

```diff
@@ -64,38 +97,92 @@ def test03_clone_rackhd_source(self):
-                                                + fit_common.GLOBAL_CONFIG['repos']['install']['rackhd']
-                                                + "/rackhd ~/rackhd"
+                                                + fit_common.GLOBAL_CONFIG['repos']['install']['rackhd']['repo']
+                                                + " ~/rackhd"
                                                 )['exitcode'], 0, "RackHD git clone failure.")
+        self.assertEqual(fit_common.remote_shell("cd ~/rackhd/" + ";git checkout "
+                                                 + fit_common.GLOBAL_CONFIG['repos']['install']['rackhd']['branch']
+                                                 )['exitcode'], 0, "Branch not found on RackHD repo.")
         # clone modules
         for repo in modules:
             self.assertEqual(fit_common.remote_shell(ENVVARS
-                                                    + "rm -rf ~rackhd/" + repo + ";"
+                                                    + "rm -rf ~/rackhd/" + repo + ";"
                                                     + "git clone "
-                                                    + fit_common.GLOBAL_CONFIG['repos']['install']['rackhd']
-                                                    + "/" + repo + " ~/rackhd/" + repo
+                                                    + fit_common.GLOBAL_CONFIG['repos']['install'][repo]['repo']
+                                                    + " ~/rackhd/" + repo
                                                      )['exitcode'], 0, "RackHD git clone module failure:" + repo)
+            self.assertEqual(fit_common.remote_shell("cd ~/rackhd/" + repo + ";git checkout "
+                                                     + fit_common.GLOBAL_CONFIG['repos']['install'][repo]['branch']
+                                                     )['exitcode'], 0, "Branch not found on module:" + repo)
 
-    def test04_run_ansible_installer(self):
+    def test03_run_ansible_installer(self):
         print "**** Run RackHD Ansible installer."
-        # install Ansible
-        self.assertEqual(fit_common.remote_shell(ENVVARS + "cd ~;apt-get -y install ansible")['exitcode'], 0, "Install failure.")
-        # run Ansible RackHD installer
         self.assertEqual(fit_common.remote_shell(ENVVARS +
                                                  "cd ~/rackhd/packer/ansible/;"
-                                                 "ansible-playbook -i 'local,' -c local rackhd_package.yml",
-                                                 timeout=600,
-                                                 )['exitcode'], 0, "Install failure.")
+                                                 "ansible-playbook -i 'local,' -c local rackhd_local.yml",
+                                                 timeout=1200,
+                                                 )['exitcode'], 0, "RackHD Install failure.")
+
+    def test04_install_network_config(self):
```

Comment:
Can this common code be pulled into fit_common?  It appears to be duplicated in both rackhd_source_install.py and rackhd_package_install.py.